tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -670,13 +666,11 @@ def test_ad_integration(
     3. SSH key for AD users is created when the property GenerateSshKeysForUsers is true;
     4. AD users can submit workloads;
     5. AD users filter out by LdapAccessFilter cannot access to the head node.
-
-    Optionally, it executes performance tests using OSU benchmarks.
     """
     if not is_directory_supported(region, directory_type):
         pytest.skip(f"Skipping the test because directory type {directory_type} is not supported in region {region}")
 
-    head_node_instance_type = "c5n.18xlarge" if request.config.getoption("benchmarks") else "c5.xlarge"
+    head_node_instance_type = "c5.xlarge"
```

Comment:
Minor: since this is now constant we can put it in the cluster config and remove the variable injection.